ItalianTranslation_PlanetCoaster.xml

```diff
@@ -20693,8 +20693,8 @@
 		<entry key="Objectives_HotelUses_View_Multiple_Any" translation="Have {Value:int} groups use your hotels a month, for {Months:int} consecutive months"/>
 		<entry key="Objectives_HotelUses_View_Multiple_Luxury" translation="Have {Value:int} groups use luxury hotel rooms a month, for {Months:int} consecutive months"/>
 		<entry key="Objectives_HotelUses_View_Multiple_Standard" translation="Have {Value:int} groups use standard hotel rooms a month, for {Months:int} consecutive months"/>
-		<entry key="Objectives_HotelUses_View_Singular_Any" translation="Have {Value:int} groups use your hotels in a month"/>
-		<entry key="Objectives_HotelUses_View_Singular_Luxury" translation="Have {Value:int} groups use luxury hotel rooms in a month"/>
+		<entry key="Objectives_HotelUses_View_Singular_Any" translation="Avere {Value:int} gruppi che usano i tuoi hotel in un mese"/>
```

Comment:
Queste due non vanno bene, falle meglio
"avere N gruppi che usano i tuoi hotel in un mese" non ha senso e quella sotto non è tradotta